content/docs/setup/kubernetes/quick-start/index.md

```diff
@@ -215,7 +215,40 @@ Nevertheless, you must update the list of admission controllers.
 
 ### Azure
 
-You must use `ACS-Engine` to deploy your cluster.
+You can deploy a Kubernetes cluster to Azure via `ACS-Engine` or `AKS` which fully supports Istio as long as Kubernetes 1.9.6 or later is specified.
+
+AKS is a managed Kubernetes service where the master nodes are paid for and managed by Microsoft, no matter the size or number of agent nodes created.  
+
+ACS-Engine provides extensive customization of the Kubernetes cluster being provisioned.  The downloadable `acs-engine` binary processes a simple api-model JSON file into a large Azure resource manager template which automatically deploys a complete Kubernetes cluster.  Master and agent nodes are mangaged entirely and paid for by the customer.
```

Comment:
Likewise. Please remove. A link should suffice if a user would like further detail.